extensions/example-extension/page.tsx

```diff
@@ -1,10 +1,10 @@
-import { Button, Icon, IconProps, LensRendererExtension } from "@lens/ui-extensions";
+import { Component, LensRendererExtension } from "@lens/ui-extensions";
```

Comment:
How to use types from common components?
I expect to consume types like this with current approach:
```
interface MyCheckbox extends Component.Checkbox.Props {}
```